test/__tests__/config-name.test.js

```diff
@@ -0,0 +1,46 @@
+"usr strict";
+
+const run = require("../testUtils");
+
+test("found-many", () => {
+	const { code, stdout, stderr } = run("config-name/found-many", [
+		"--config", "./webpack.config.js",
```

Comment:
it might make sense to pass an object which we turn into an array instead of the array directly. Easier to apply defaults, then. Up to you, though!